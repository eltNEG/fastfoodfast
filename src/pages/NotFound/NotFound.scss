.page-not-found {
    background: url('../../assets/images/Screenshot\ 2019-03-11\ at\ 5.41.32\ AM.png') no-repeat center;
    background-size: cover;
    height: 14rem;
    width: 14rem;

}

.page-not-found-container {
    display: flex;
    justify-content: center;
    align-content: center;
    align-items: center;
    position: absolute;
    height: 80vh;
    width: 100%;
    background-color: transparent;
}