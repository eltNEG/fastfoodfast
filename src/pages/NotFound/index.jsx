import React from 'react';
import './NotFound.scss';

export default () => (
  <div className="page-not-found-container">
    <div className="page-not-found" />
  </div>
);
