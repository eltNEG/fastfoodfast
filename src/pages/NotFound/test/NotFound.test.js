import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '..';

describe('NotFound snapshot', () => {
  it('should match NotFound.jsx snapshot', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});