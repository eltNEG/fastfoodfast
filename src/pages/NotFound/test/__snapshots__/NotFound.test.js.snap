// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound snapshot should match NotFound.jsx snapshot 1`] = `
<div
  className="page-not-found-container"
>
  <div
    className="page-not-found"
  />
</div>
`;
